Kernel/Syscalls/mmap.cpp

```diff
@@ -567,9 +569,11 @@ ErrorOr<FlatPtr> Process::sys$allocate_tls(Userspace<char const*> initial_data,
         auto& tls_descriptor = Processor::current().get_gdt_entry(GDT_SELECTOR_TLS);
         tls_descriptor.set_base(main_thread->thread_specific_data());
         tls_descriptor.set_limit(main_thread->thread_specific_region_size());
-#else
+#elif ARCH(X86_64)
         MSR fs_base_msr(MSR_FS_BASE);
         fs_base_msr.set(main_thread->thread_specific_data().get());
+#elif ARCH(AARCH64)
+        TODO_AARCH64();
```

Comment:
pls annotate this todo to say that we should be setting thread-local data into tpidr_el0